examples/guestbook/README.md

```diff
@@ -139,7 +216,7 @@ redis-master-dz33o                             1/1       Running   0          2h
 `kubectl get pods` will show only the pods in the default [namespace](../../docs/user-guide/namespaces.md).  To see pods in all namespaces, run:
 
 ```
-kubectl get pods -o wide --all-namespaces=true
+kubectl get pods -o wide --all-namespaces
```

Comment:
Add references to:

https://github.com/kubernetes/kubernetes/blob/master/docs/user-guide/configuring-containers.md
https://github.com/kubernetes/kubernetes/blob/master/docs/user-guide/deploying-applications.md